NewDawn/NewDawn/ViewControllers/RegistrationViewControllers/Profile_HomeTown.swift

```diff
@@ -18,29 +18,30 @@ class Profile_HomeTown: UIViewController {
     
     /* Constrains */
     
-    var visibleField = false
+//    var visibleField = false
 
     
     @IBOutlet weak var hometownTextField: UITextField!
```

Comment:
In order to move the text, you need to create an outlet for continueButton and move up the title in the button. Take a look at viewDidLoad() in ProfileGNBViewController as a reference.